scripts/dumpRobloxTypes.py

```diff
@@ -637,6 +637,31 @@
 # More hardcoded types, but go at the end of the file
 # Useful if they rely on previously defined types
 END_BASE = """
+declare Vector2: {
```

Comment:
Do we need to add an ignore somewhere given that we have defined an inlined definition instead?

Reply:
it seems to work fine without any ignores since we declare at the end, it overrides other declares of vector2/vector3
rest of the types in that declare (other than min/max) is exactly same as dump

Reply:
i think we should probably put an explicit ignore for clarity